src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
```

Comment:
I don't see this method here
![Screenshot 2022-08-18 at 09 40 02](https://user-images.githubusercontent.com/60258373/185313244-1145023e-5699-4802-95be-b0c1953eed99.png)